src/Network/Network/ChangeLog.md

```diff
@@ -36,6 +36,7 @@
 * Fixed a few minor issues
 * Onboarded `Microsoft.DBforPostgreSQL/flexibleServers` to private link cmdlets
 * Fixed missing properties in PSBackendAddressPool
+* Added cmdlet `Convert-AzNetworkWatcherClassicConnectionMonitor` for converting a classic connection monitor to V2 connection monitor..
```

Comment:
The changelog should be added below "upcoming release" for new release

Reply:
Resolved

Reply:
Still not resolved. Place it under "##upcoming release“ and please remove two dot at the end of this line.